web_development_101/project_git_basics.md

```diff
@@ -21,16 +21,16 @@ By the end of this you should be able to:
 Here we will describe the basic Git workflow that you will use with your projects, follow along with each of the steps.
 
 #### Create your repository on Github
-1. You should have created a GitHub account in the [installations](https://www.theodinproject.com/courses/web-development-101/lessons/installations) project. If you haven't done that yet you can sign up [here](https://github.com/).
+1. You should have created a GitHub account in the [installation prerequisites](https://www.theodinproject.com/courses/web-development-101/lessons/prerequisites) project. If you haven't done that yet you can sign up [here](https://github.com/).
```

Comment:
https://www.theodinproject.com/courses/web-development-101/lessons/your-first-rails-application?ref=lnav

The github account was created here ^

Reply:
Wow good catch. 

Adjusting it now.